chart/templates/_helpers.tpl

```diff
@@ -926,6 +926,39 @@ Add environment variables to configure database values
 {{- ternary "5432" .Values.externalPostgresql.port .Values.internalPostgresql.enabled | quote -}}
 {{- end -}}
 
+{{/*
+Get the Postgresql config map name
+*/}}
+{{- define "carto.postgresql.configMapName" -}}
+{{- if .Values.internalPostgresql.enabled -}}
+  {{- printf "%s-%s" (include "carto.postgresql.fullname" .) "ssl-ca" -}}
+{{- else }}
+  {{- printf "%s-%s-%s" .Release.Name "externalpostgresql" "ssl-ca" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the directory where the Postgresql CA cert will  be mounted
+*/}}
+{{- define "carto.postgresql.secretMountDir" -}}
```

Comment:
This name is a bit confused. I was thinking all the time that the CA is going to be treated as a secret and finally it was a configmap :see_no_evil: .

Maybe we should rename that to something like... ?
```suggestion
{{- define "carto.postgresql.caMountDir" -}}
```

Reply:
We are doing this way with other similar variables

```
carto.gcpBucketsServiceAccountKey.secretMountDir
carto.google.secretMountDir
```

Reply:
But they are "Secrets". Maybe we can leave at:
 ```suggestion
{{- define "carto.postgresql.configMapMountDir" -}}
```
or similar